GameServer/Logic/GameObjects/GameObjects.cs

```diff
@@ -40,7 +40,8 @@ public void SetTeam(TeamId team)
         public int CollisionRadius { get; set; }
         private Vector2 _direction;
         public int VisionRadius { get; private set; }
-        public bool IsDashing { get; private set; }
+        public bool IsDashing { get; protected set; }
+        public bool IsDashingToTarget { get; protected set; }
```

Comment:
This property to me seems like it's not used anywhere. Is it really necessary?

Also if it is necessary, wouldn't it be smarter to add a `DashTarget` property instead? Now you're just doing `if (IsDashing || IsDashingToTarget)` everywhere, which seems like it could be done simpler.

I'd suggest you to remove the `IsDashingToTarget` property, and use a `DashTarget` property instead. Then if you actually need to check for targetted dashes somewhere, you can check `if (DashTarget != null)`